sdk/metricsadvisor/Azure.AI.MetricsAdvisor/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Breaking Changes
 - The whole `DatasourceCredential` API has been renamed to `DataSourceCredential`. This includes renames in types, methods, and properties.
+- Split `GetAnomalies` overloads into two different methods: `GetAnomaliesForAlert` and `GetAnomaliesForDetectionConfiguration`.
```

Comment:
Because there are no `GetAnomalies` methods remaining, I will suggest changing this to:
`Split the method GetAnomalies into two different methods.....`